package.json

```diff
@@ -21,6 +21,9 @@
   "dependencies": {
     "@istanbuljs/nyc-config-typescript": "^1.0.2",
     "@types/multer": "^1.4.11",
+    "@types/nodemailer": "^6.4.14",
+    "@types/randomstring": "^1.3.0",
+    "@types/sinon": "^17.0.3",
```

Comment:
@kayigmb type definition dependencies should be installed as devDependencies. Move types under devDependencies.